src/main/resources/swagger.api/schemas/tenant.yaml

```diff
@@ -5,8 +5,13 @@ Tenant:
       type: string
     code:
       type: string
```

Comment:
'code' and 'name' are unique, I think we also need to verify that

Reply:
Thanks to remind, I will look at